### ISSUE TITLE
Save the whole form to one cookie?

### ISSUE THREAD

**Issue by crabnky** [awhipp/inputStore.jquery#2]
Hi,
is there a possibility to have one cookie holding data for the whole form?
Cheers

**Comment by awhipp** [comment]
@crabnky 

> Hi,
> is there a possibility to have one cookie holding data for the whole form?
> Cheers

Yeah that's entirely possible. Just have a json array saved in the cookie. Feel free to work on it. If you put a pull request in Ill review and accept it.
